zh_cn.json

```
@@ -0,0 +1,139 @@
+{
+  "fluid_type.brewinandchewin.honey_type": "蜂蜜",
+  "fluid_type.brewinandchewin.beer_type": "啤酒",
+  "fluid_type.brewinandchewin.vodka_type": "伏特加",
+  "fluid_type.brewinandchewin.mead_type": "蜂蜜酒",
+  "fluid_type.brewinandchewin.egg_grog_type": "鸡蛋格罗格",
+  "fluid_type.brewinandchewin.strongroot_ale_type": "强根麦芽酒",
+  "fluid_type.brewinandchewin.rice_wine_type": "米酒",
+  "fluid_type.brewinandchewin.glittering_grenadine_type": "发光果浆",
```

```suggestion
  "fluid_type.brewinandchewin.glittering_grenadine_type": "闪耀果浆",
```
发光（Glowing）和闪烁（Glistering）在原版中均有出现，不宜作为Glittering的对应，考虑使用与闪烁（Glistering）相近的“闪耀”以体现原文Glittering与Glistering相近的特点。